src/components/batch-selection/batch-selection.pw.tsx

```diff
@@ -53,7 +53,7 @@ test.describe("check BatchSelection component properties", () => {
     await mount(<BatchSelectionComponent hidden />);
     const batchSelection = batchSelectionComponent(page);
     await expect(batchSelection).toHaveAttribute("hidden", /.*/);
-    await expect(batchSelection).toHaveCSS("opacity", "0");
+    await expect(batchSelection).toHaveCSS("visibility", "hidden");
```

Comment:
**suggestion**: `.not.toBeVisible()` may be more future-proof than asserting against the attribute